main/lib/webApi.js

```diff
@@ -1,46 +1,39 @@
 import request from 'superagent'
 import { Dispatcher } from 'fluxxed_up'
 
-export function fetch(url, data, actionType) {
-  buildRequest(request.get(url), null, actionType)
-}
-
-export function post(url, data, actionType) {
-  buildRequest(request.post(url), data, actionType)
+function getErrors(res) {
```

Comment:
@bartriordan thanks for the edits here, but I was actually gonna just integrate `jsonStatham` and remove `webApi` now that we have a working implementation of `adaptor`.  The key with `webApi` was that it allowed me to pass the `accessToken` a.k.a. `authToken` before we fixed up that aspect of `fluxxed_up`.